photo-checkin/src/photo_checkin.py

```diff
@@ -1,14 +1,11 @@
-import json
 import urllib.parse
 import boto3
 
 print('Loading function')
 
 s3 = boto3.client('s3')
 
-def lambda_handler(event, context):
-    # print("Received event: " + json.dumps(event, indent=2))
-
+def handler(event, context):
     # Get the object from the event and show its content type
     bucket = event['Records'][0]['s3']['bucket']['name']
     key = urllib.parse.unquote_plus(event['Records'][0]['s3']['object']['key'], encoding='utf-8')
```

Comment:
you could get multiple records. therefore use a for loop

Reply:
ok